### PR TITLE
Issue 5020: Change slack link on README to point to the sign-up page

### LINKED ISSUE #5020: README Support section has outdated contact info
**Problem description**
README Support section links to defunct mailing list and omits Slack signup/invite page.

**Problem location**
https://github.com/pravega/pravega/blob/c7ac009970787df2633163644691aa3d72fc1a4e/README.md#support

**Suggestions for an improvement**
Remove link to defunct mailing list, add Slack signup/invite page alongside Slack workspace URL.

### PR DESCRIPTION
**Pull request by rofr**
** [UPDATE] **
Add a link to the slack signup page but also keep the existing link

**Change log description**
Change the slack link under the support section to point at the sign-up page instead of the login page which has no information on how to join the slack.

**Purpose of the change**  
Make it easier for new community members to onboard.
Fixes #5020

**What the code does**  
n/a

**How to verify it**  
Follow the link in the updated README

### CODE REVIEW

**Comment by rofr** [comment]
> https://pravega-io.slack.com/ is the correct slack space once you're a member, so including both links is probably ideal.
> 
> It seems we've abandoned using Google Groups, and we will be getting mailing lists from the open source foundation we incubate with, so we should also remove the Google Groups link.

PR updated:
* Keep the link to the slack in addition to adding a link to the signup page
* Removed the Google Groups reference as well
* Fixed a small typo
* Squashed to a single commit

File: README.md [hunk 1]
```diff
@@ -71,13 +71,11 @@ Pravega can be installed locally or in a distributed environment. The installati
 
 ## Support
 
-Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://pravega-io.slack.com/) if you need any help.
-Open an issue if you found a bug on [Github
-Issues](https://github.com/pravega/pravega/issues).
+Don’t hesitate to ask! Contact the developers and community on [slack](https://pravega-io.slack.com/) ([signup](https://pravega-slack-invite.herokuapp.com/)) if you need any help. Open an issue if you found a bug on [Github Issues](https://github.com/pravega/pravega/issues).
```
Comment (fpj) [inline README.md:74]: Looks good. One question: would it make sense to have both a "login here" and "sign up here"?

Comment (rofr) [reply README.md:74]: Both links are there? The slack link brings you to the login page.

**Comment by RaulGracia** [comment]
@fpj can we merge this one?

**Comment by andreipaduroiu** [comment]
Are there any updates on this PR? I've seen no response to my question from 2 months ago. If I don't hear back, I will close this PR.

**Comment by derekm** [comment]
@andreipaduroiu -- I've created the issue & updated the PR for @rofr. Thanks!

**Pull request merged**